heralding/reporting/file_logger.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, logFile):
             self.fileHandler = open(logFile, 'a')
 
         fieldNames = ['timestamp', 'auth_id', 'session_id', 'source_ip', 'source_port', 'destination_ip',
-                      'destination_port', 'protocol', 'username', 'password']
+                      'destination_port', 'protocol', 'auth_mech', 'username', 'password']
```

Comment:
This is a breaking change, how about we keep the auth_mech stuff internal for now (maybe log it with log.debug...)?